package.json

```diff
@@ -292,7 +295,15 @@
 		"@automattic/newspack-blocks/prettier": "npm:wp-prettier@2.6.2-beta-1",
 		"@types/react": "^17.0.39",
 		"keytar@npm:7.7.0/node-addon-api": "3.1.0",
-		"lzma-native": "^8.0.5"
+		"lzma-native": "^8.0.5",
+		"abab": "^2.0.6",
+		"acorn-walk": "^8.2.0",
+		"decimal.js": "^10.4.1",
+		"https-proxy-agent": "^5.0.1",
+		"nwsapi": "^2.2.2",
+		"tough-cookie": "^4.1.2",
+		"ws": "^8.9.0",
+		"node-gyp-build": "^4.2.1"
```

Comment:
I noticed we aren't importing most of these dependencies. Are they all peer dependencies of jsdom?

Reply:
Indeed, these were needed because the duplicate packages were failing the TC  at the Prevent duplicated packages step. I added these based on the `yarn dedupe --check` output.
I reran it and the list has shrunk now to just `decimal.js`, `node-gyp-build` and `psl`.

Reply:
Oh, interesting. Typically, you just need to run `yarn dedupe`, and it updates the lockfile. The goal is to make sure that the transitive dependencies use the same version wherever possible. Normally, you don't need to update package.json. (Typically, you only need to update package.json if we explicitly import it, or it's explicitly a peer dependency.)